src/main/java/co/infinum/complexify/Complexify.java

```diff
@@ -188,14 +190,15 @@ public Complexify() {
     /**
      * Constructor
      *
-     * @param banMode             Use strict or loose comparisons for banned passwords. (default: ComplexifyBnaMode.STRICT)
+     * @param banMode             Use strict or loose comparisons for banned passwords. (default: ComplexifyBanMode.STRICT)
      * @param strengthScaleFactor Required password strength multiplier (default: 1)
      * @param minimumChars        Minimum password length (default: 8)
      */
     public Complexify(ComplexifyBanMode banMode, int strengthScaleFactor, int minimumChars) {
         this.banMode = banMode;
```

Comment:
won't be much of a trouble, but it would break backward compatibility. are you ok with that?